show what is not funded

lisbethmarianne opened the pull request:
Issue #218 

![bildschirmfoto 2016-07-03 um 13 55 45](https://cloud.githubusercontent.com/assets/11691539/16545277/f2a110f8-4125-11e6-9f6b-081904b25216.png)

svenfuchs commented:
How about using a css class for prepending the icon ... this would then also allow to dim down the color. Something like:

```
# views/events/_event.html.erb

<%= span_for_funded('Ticket', @event.ticket_funded) %>
<%= span_for_funded('Accomodation', @event.accomodation_funded) %>
<%= span_for_funded('Travel', @event.travel_funded) %>

# helpers/something.rb

def span_for_funded(title, funded)
  content_tag(:span, title, class: funded ? 'funded' : 'not-funded')
end

# stylesheets/something.css

.funded:before {
  content: "\u2714"
}

.not-funded:before {
  content: "\u2718"
}

.not-funded {
  color: #aaa;
}
```

I'm not 100% sure about the css, but this is how i remember it works :)

lisbethmarianne commented:
Thanks for the suggestion @svenfuchs, looks like this now:

![bildschirmfoto 2016-07-09 um 11 21 24](https://cloud.githubusercontent.com/assets/11691539/16707244/ab1f4622-45c7-11e6-9e26-02b11a1d830f.png)

(I noticed that events.scss was never loaded and fixed that, also deleted 2 unused scss files.)
